### ISSUE TITLE
Improve summary statistics

### ISSUE THREAD

**Issue by charlie-hall20** [george0angel/drunkbird#6]
- statistics should be more mathematically named 
- alt text to more detail
-         e.g. "All-Time Maximum Position" with alt-text "max { X_s(u) : 0 \le s \le t , u\in \mathcal{N}_t  }"
-         e.g. "End-Time Maximum Position" with alt-text "max { X_t(u) : u\in \mathcal{N}_t  }"

I can provide a list of relevant statistics with the maths definitions when we get round to this - charlie

**Comment by george0angel** [comment]
If you let me know which statistics you think should be removed or added then I can put them in and I'll have a way for you to write alt text for them later?

**Comment by charlie-hall20** [comment]
**Will come back to edit this list when I have time** 

For one-dim. Brownian Motion:
- Maximum position at end-time ($\max { X_t(u) : u\in \mathcal{N}_t  }$)
- Minimum position at end-time ($\min { X_t(u) : u\in \mathcal{N}_t  }$)
- All-time maximum position ($\max { X_s(u) : u\in \mathcal{N}_t , s \le t }$)
- All-time minimum position ($\min{ X_s(u) : u\in \mathcal{N}_t , s \le t }$)
- Time of all-time maximum
- Population Size at end time ($|\mathcal{N}_t|$)
- Mean final position ($\frac{1}{|\mathcal{N}_t|}\sum_{ u\in \mathcal{N}_t}  X_t(u)$)
- First Branch Time 

For higher dimensions: 
- Maximum displacement from origin at end-time ($\max { |X_t(u)| : u\in \mathcal{N}_t  }$)
- All-time Maximum displacement from origin ($\max { |X_s(u)| : u\in \mathcal{N}_t , s \le t }$)
- Time of all-time maximum displacement
- Population Size at end time ($|\mathcal{N}_t|$)
- Mean final position ($\frac{1}{|\mathcal{N}_t|}\sum_{ u\in \mathcal{N}_t}  X_t(u)$)
- First Branch Time 

For Random Walk:
**Basically all the same as above** as well as

- no of times the origin is visited
- in 1D, the proportion of time the walk is positive

**Comment by george0angel** [comment]
I have added the new summary statistics, see 84bfba23214689d018e22007196979082ac2af06. Are you sure we need tooltips for all of these?
